docs/esa/iso/iso.rst

```diff
@@ -119,75 +115,86 @@ observation identifier (TDT) as input. There are different product levels:
+  ...                               "ida.observations WHERE " +
   ...                               "revno=800 and aotname like 'S%' and target != ''")
-  >>> print(table)
+  >>> print(table)   # doctest: +IGNORE_OUTPUT
     tdt    revno aotname      ra        dec   
                               h         deg   
   -------- ----- ------- ------------ --------
-  80002504   800    S01       2.74033 55.18761
-  80002450   800    S01      2.550044 58.03472
-  80002304   800    S07   2.428013333 62.09789
   80002247   800    S07   2.428012666 62.09789
   80002014   800    S02   0.901055333 73.08528
-  80000938   800    S01     21.691944 76.37833
+  80002504   800    S01       2.74033 55.18761
   80000828   800    S02  21.035861333 68.15064
   80000104   800    S01  16.477365333 41.88163
+  80000938   800    S01     21.691944 76.37833
+  80002304   800    S07   2.428013333 62.09789
+  80002450   800    S01      2.550044 58.03472
+  >>>
   >>> for observation in table:
-  >>>     ISO.download_data(str(observation['tdt']), retrieval_type="OBSERVATION", 
-  ...        product_level="DEFAULT_DATA_SET", filename=str(observation['tdt']), verbose=True)
-  INFO: http://nida.esac.esa.int/nida-sl-tap/data? 
-  retrieval_type=OBSERVATION&tdt=80002504&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
+  ...     ISO.download_data(str(observation['tdt']), retrieval_type="OBSERVATION",
+  ...        product_level="DEFAULT_DATA_SET", filename=str(observation['tdt']), verbose=True)  # doctest: +SKIP
+  INFO: http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80000104&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
+  INFO: Copying file to 80000104.tar... [astroquery.esa.iso.core]
+  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80000104&product_level=DEFAULT_DATA_SET to 80000104.tar [astroquery.esa.iso.core]
+  '80000104.tar'
+  INFO: http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002504&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
   INFO: Copying file to 80002504.tar... [astroquery.esa.iso.core]
-  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data? 
-  retrieval_type=OBSERVATION&tdt=80002504&product_level=DEFAULT_DATA_SET to 80002504.tar [astroquery.esa.iso.core]
-  INFO: http://nida.esac.esa.int/nida-sl-tap/data? 
-  retrieval_type=OBSERVATION&tdt=80002450&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
+  INFO: Wrote http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80002504&product_level=DEFAULT_DATA_SET to 80002504.tar [astroquery.esa.iso.core]
+  '80002504.tar'
+  INFO: http://nida.esac.esa.int/nida-sl-tap/data?retrieval_type=OBSERVATION&DATA_RETRIEVAL_ORIGIN=astroquery&tdt=80000938&product_level=DEFAULT_DATA_SET [astroquery.esa.iso.core]
```

Comment:
try to hide all these INFO in an ellipsis, after all we skip the line above. 

(but again, maybe there is a smaller example that can be put in here and be run? cc @jespinosaar)